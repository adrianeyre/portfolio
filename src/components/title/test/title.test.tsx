import React from 'react';
import { shallow } from 'enzyme';

import ITitleProps from '../interface/title-props';
import Title from '../title';

describe('Image', () => {
	const defaultProps: ITitleProps = {
		title: 'My Title',
		subTitle: 'My Subtitle',
	};
	
	it('Should render correctly', () => {
		const imageComponent = shallow(<Title { ...defaultProps }/>);

		expect(imageComponent).toMatchSnapshot();
	});
});
