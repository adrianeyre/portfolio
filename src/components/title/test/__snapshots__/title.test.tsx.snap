// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image Should render correctly 1`] = `ShallowWrapper {}`;
